package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "kontist",
-  "version": "0.36.13",
+  "version": "0.36.14",
```

Comment:
❌ Please don't update version number in the feature PR, because it would cause problems if there is somebody else who is working on the SDK.
Version number should be bumped in a separate PR after this one is merged